docs/root/index.md

```diff
@@ -63,14 +64,13 @@ GA:
   - Additionally, the following libraries are available for access to beta API functionality:
     - [Google.Cloud.Vision.V1P1Beta1](Google.Cloud.Vision.V1P1Beta1/index.html)
     - [Google.Cloud.Vision.V1P2Beta1](Google.Cloud.Vision.V1P2Beta1/index.html)
+- [Grafeas.V1](Grafeas.V1/index.html)
```

Comment:
The indent seems to be wrong.

Reply:
Nope, it's just that DLP just above has a two-level-nested list. It's the same as other top-level packages.